staging/com.scoreSmart/manifest.tt

```diff
@@ -0,0 +1,27 @@
+class @com.scoreSmart {
+  import loader from @org.thingpedia.v2();
+  import config from @org.thingpedia.config.none();
+
+  monitorable list query standings(in req league_id: String #_[prompt="what league standings do you want out check out?"],
+                        out Team: String,
+                        out Rank: Number,
+                        out Form: String,
+                        out Points: Number)
```

Comment:
Identifiers that start with an upper-case letter in ThingTalk are reserved and better avoided.